daemon/src/maker.rs

```diff
@@ -270,9 +272,27 @@ async fn main() -> Result<()> {
         },
         time::Duration::hours(opts.settlement_time_interval_hours as i64),
         N_PAYOUTS,
+        projection_actor.clone(),
     )
     .await?;
 
+    let (task, init_quote) = bitmex_price_feed::new(projection_actor).await?;
+    tasks.add(task);
+
+    let cfds = load_all_cfds(&mut conn).await?;
```

Comment:
It's not expensive to do this, but now we load all cfds twice upon startup - here and in the actor system in `lib.rs`.

Reply:
Right... I don't think this is an issue here or do you have any concerns?

Reply:
Let's fix this in a follow up if you think this is important.